org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/rewrite/ImportRewrite.java

```diff
@@ -393,13 +392,20 @@ public static ImportRewrite create(ICompilationUnit cu, boolean restoreExistingI
 	 * direct and transitive imports.
 	 *
 	 * @param binding - module binding
-	 * @param currentModuleName - name of the module for this program or null
+	 * @param cu - compilation unit
```

Comment:
I can see that passing `cu` is convenient, but your lack of explanation tells me, that it is not trivial to explain why a cu is relevant for this method. Which cu should clients pass? 

What's easier for you: to clearly motivate the role of the `cu` parameter, or change the type to something that can be easier explained? IMHO, any subexpression of `cu.getJavaElement().getJavaProject().getModuleDescription()` would equally qualify as the value to be provided by callers.

Note, that this is API and should be done considering also clients outside JDT.

Reply:
go ahead :)